crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/model/KycRefreshRequest.kt

```diff
@@ -0,0 +1,68 @@
+package com.stripe.android.crypto.onramp.model
+
+import androidx.annotation.RestrictTo
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * Internal API request model for KYC refreshing.
+ * This represents the exact structure expected by the Stripe API.
+ */
+@Serializable
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+internal data class KycRefreshRequest(
+    @SerialName("first_name")
+    val firstName: String,
+    @SerialName("last_name")
+    val lastName: String,
+    @SerialName("id_number_last4")
+    val idNumberLastFour: String?,
+    @SerialName("id_type")
+    val idType: String?,
+    @SerialName("dob")
+    val dateOfBirth: DateOfBirth,
+    @SerialName("city")
+    val city: String?,
+    @SerialName("country")
+    val country: String?,
+    @SerialName("line1")
+    val line1: String?,
+    @SerialName("line2")
+    val line2: String?,
+    @SerialName("zip")
+    val postalCode: String?,
```

Comment:
Per the doc, shouldn't this be `postal_code`?

```suggestion
    @SerialName("postal_code")
    val postalCode: String?,
```

Either way, the serialized string should match the symbol name

Reply:
According to the conversation that was had [here](https://lickability.slack.com/archives/C094P3BRBL1/p1761668312557619?thread_ts=1761597746.511609&cid=C094P3BRBL1), and I also believe in the linked PRD spec, the serialized name for the request is indeed `"zip"`, while the response for the refresh is `"postal_code"`, and that is intentional.

I can rename the symbol, but this pattern does match the existing KYCCollectionRequest, and this from searching would be the first time we would use `zip` as a symbol name. Let me know if you prefer that.

Reply:
Got it, let's leave it as is then